File: Leetcode/string/1078_OccurrencesAfterBigram/Solution.java
class Solution {
    public String[] findOcurrences(String text, String first, String second) {
        String[] strs = text.split(" ");
        List<String> res = new ArrayList<>();
        for(int i = 0; i<strs.length-2; ++i){
            if( strs[i].equals(first) && strs[i+1].equals(second) ){
                res.add(strs[i+2]);
            }
        }
        return res.toArray(new String[0]);
    }
}

File: Leetcode/string/1078_OccurrencesAfterBigram/Solution.py
class Solution(object):
    def findOcurrences(self, text, first, second):
        """
        :type text: str
        :type first: str
        :type second: str
        :rtype: List[str]
        """
        if not text:
            return None
        res = []
        strs = text.split(" ")
        for i in range(0, len(strs)-2):
            if strs[i]==first and strs[i+1]==second:
                res.append(strs[i+2])
        return res
